Exam_10.js

```diff
@@ -0,0 +1,215 @@
+// Bài 1: in một bảng nhân của một sô bất kì với các số từ 1 đến 10 và hiện kết quả
+const multiplyNumber = (n) => {
+        let result
+        for (let i = 1; i < 11; i++) {
+            result = n * i
+            console.log(`${n} x ${i} = ${result}`)
+        }
```

Comment:
```suggestion
        for (let i = 1; i < 11; i++) {
            const result = n * i
            console.log(`${n} x ${i} = ${result}`)
        }
```
Biến result đặt bên ngoài thì mang ý nghĩa có thể dùng bên ngoài vòng for, nhưng khi kết thúc vòng for không có lệnh nào dùng cả -> chuyển vào trong vòng for